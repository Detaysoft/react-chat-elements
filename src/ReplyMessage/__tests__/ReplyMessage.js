import React, { Component } from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import ReplyMessage from '../ReplyMessage'

describe('ReplyMessage component', () => {
  it('should render without issues', () => {
    const component = shallow(<ReplyMessage titleColor='' />)
    expect(component.length).toBe(1)
    expect(toJson(component)).toMatchSnapshot()
  })
})
