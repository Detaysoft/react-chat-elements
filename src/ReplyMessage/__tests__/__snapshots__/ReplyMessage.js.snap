// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReplyMessage component should render without issues 1`] = `
<div
  className="rce-mbox-reply"
  style={Object {}}
>
  <div
    className="rce-mbox-reply-left"
  >
    <div
      className="rce-mbox-reply-owner"
      style={Object {}}
    >
      Unknown
    </div>
    <div
      className="rce-mbox-reply-message"
    >
      ...
    </div>
  </div>
</div>
`;
